platform_umbrella/apps/common_core/lib/common_core/postgres/backup_config.ex

```diff
@@ -0,0 +1,17 @@
+defmodule CommonCore.Postgres.BackupConfig do
```

Comment:
The other embedded structs are prefixed with PG should we do that here to follow, or should we re-name others since they are already in the Postgres namespace?